lib/takeaway.rb

```diff
@@ -1,2 +1,10 @@
+require_relative 'menu.rb'
+
 class Takeaway
+
+  def show_menu
+    menu_person = Menu.new
```

Comment:
what does menu_person mean? perhaps a more obvious variable name could be used here?